docs/api/covidcast-signals/google-symptoms.md

```diff
@@ -19,23 +19,45 @@ grand_parent: COVIDcast Epidata API
 
 This data source is based on the [COVID-19 Search Trends symptoms
 dataset](http://goo.gle/covid19symptomdataset). Using
-this search data, we estimate the volume of searches mapped to symptoms related
-to COVID-19 such as _anosmia_ (lack of smell) and _ageusia_(lack of taste). The
-resulting daily dataset for each region shows the relative frequency of searches
-for each symptom. The signals are measured in arbitrary units that are
-normalized for overall search users in the region and scaled by the maximum value of the normalized
-popularity within a geographic region across a specific time range. **Thus,
-values are NOT comparable across geographic regions**. Larger numbers represent
-increased releative popularity of symptom-related searches.
+this search data, we estimate the volume of searches mapped to symptom sets related
+to COVID-19. The resulting daily dataset for each region shows the average relative frequency of searches for each symptom set. The signals are measured in arbitrary units that are normalized for overall search users in the region and scaled by the maximum value of the normalized popularity within a geographic region across a specific time range. **Thus, values are NOT comparable across geographic regions**. Larger numbers represent increased releative popularity of symptom-related searches.
```

Comment:
this would be new but occurred to me after your talk on Friday -- thoughts?
```suggestion
to COVID-19. The resulting daily dataset for each region shows the average relative frequency of searches for each symptom set. The signals are measured in arbitrary units that are normalized for overall search users in the region and scaled by the maximum value of the normalized popularity within a geographic region across a specific time range. **Values are comparable across signals but NOT across geographic regions**. Larger numbers represent increased relative popularity of symptom-related searches.
```

Reply:
I like the update! I'll add an extra sentence to clarify.